packages/fxa-shared/email/validateEmail.js

```diff
@@ -0,0 +1,32 @@
+const dns = require('dns');
+const resolver = new dns.promises.Resolver();
+const NotFoundErrorCodes = require('./emailValidatorErrors').NotFoundErrorCodes;
```

Comment:
Mixing up variable declarations with imports doesn't follow our style guidelines (which are poorly defined) or existing code conventions.

Reply:
I have updated this to `import { NotFoundErrorCodes } from './emailValidatorErrors';` let me know if that rectifies  your issue with this line.

just a heads-up (also to clarify in case I am misunderstanding this) we do have the following in our code:
```
const release = require('../../package.json').version;
const config = require('../config').getProperties();
const METRICS_CONTEXT_SCHEMA = require('../metrics/context').schema;
const jwtverify = require('util').promisify(jsonwebtoken.verify);
const hex = require('buf').to.hex;
```
i am not sure if these examples are similar or different enough, so pardon my ignorance.

i've brought up this issue with anna before, is there a style guide you can reference?  it seems our we often have mixed conventions and then it comes during code review that something is against convention when our code base isnt _seemingly_ consistent enough to tease this out during development time. 

let me know if the other examples i posted arent what you meant, i am trying to learn how to avoid these mistakes going forward.